cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         volumeMounts:
           - name: config
             mountPath: /etc/config
+      nodeSelector:
+        beta.kubernetes.io/masq-agent-ds-ready: "true"            
```

Comment:
Right but if those env vars are set in GKE that would also handle this.

Yesterday a PR that set NON_MASQUERADE_CIDR=0.0.0.0/0 in the gce scripts was reverted.  Since that was reverted the non-masq-cidr will get set to 10/8.  If that is the case the the ip-masq-agent won't even run and the kubelet would take care of the masquerading.  So I think that whether those labels are set here doesn't matter.

I think what is needed now is that we set gce back to using NON_MASQUERADE_CIDR=0.0.0.0/0 and we add the labels.